README.md

```diff
@@ -59,10 +60,11 @@ See the file [COPYING](COPYING) for more details.
 
 HISTORY
 -------
+XQF was originally written by Roman Pozlevich from Pereslavl-Zalessky, Russia. Over 17 years of existence the project has had the following major developers:
```

Comment:
is Roman agrees with this public information: “from Pereslavl-Zalessky” ? :-)
If it's true that the former creator was Russian, this information is not very important compared to the list of contributors and maintainers, and put into perspective with the age of the project.

In fact the most prolific developper is @lnussel (Germany?) and the most faithful maintainer is @jmallach (Spain/Cataluña?). If we were to specify the location of each, we will never finish. So I think this information is irrelevant.

I like the “Over 17 years of existence”, but we must not rewrite the text every year. So a date is the best way.

This whole line can't be merged at all.

I like the idea of recall his involvement in the project and his initiative, but not this way.